test-toolkit/docker/1.5.1/Dockerfile.dlc.eia

```diff
@@ -0,0 +1,6 @@
+ARG region
+FROM 763104351884.dkr.ecr.$region.amazonaws.com/mxnet-inference-eia:1.5.1-cpu-py3
```

Comment:
I think this file will also be added after the EIA image is officially released.

> Step 2/5 : FROM 763104351884.dkr.ecr.$region.amazonaws.com/mxnet-inference-eia:1.5.1-cpu-py3
---------------------------- Captured stderr setup -----------------------------
no previously-included directories found matching 'test'
warning: no previously-included files matching '__pycache__' found under directory '*'
warning: no previously-included files matching '*.py[co]' found under directory '*'
warning: check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too

WARNING! Using --password via the CLI is insecure. Use --password-stdin.
WARNING! Your password will be stored unencrypted in /root/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

manifest for 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-inference-eia:1.5.1-cpu-py3 not found: manifest unknown: Requested image not found

Reply:
Make sense